src/js/test/index.test.js

```diff
@@ -0,0 +1,105 @@
+import TestUtils from 'js-test-buddy';
+import { getBreakpoints, currentScreenWidth } from '../';
+
+describe('getBreakpoints', () => {
+
+    beforeEach(() => {
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                display: none;
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+    });
+
+    it('should return an object', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(typeof breakpoints).toBe('object');
+    });
+
+    it('should return multiple breakpoints', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(Object.entries(breakpoints).length > 1);
+    });
+
+    it('should return breakpoints with expected names', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(breakpoints.narrow).not.toBeUndefined();
+        expect(breakpoints.mid).not.toBeUndefined();
+        expect(breakpoints.wide).not.toBeUndefined();
+        expect(breakpoints.huge).not.toBeUndefined();
+    });
+
+    it('should return breakpoints in pixels', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(breakpoints.narrow.slice(-2)).toBe('px');
```

Comment:
Just test for the full value i.e. `414px` rather than calling slice throughout.

Reply:
This also means I can remove the next test 🕺